adm_plugins/calendar/config.php

```diff
@@ -43,6 +43,9 @@
 // Anzeige der Geburtstage mit Icon (1) oder ohne Icon (0)
 $plg_geb_icon = 1;
 
+// Anzeige des "Nachname, Vorname" (0) oder "Vorname" (1) andere Werte erzeugen "Nachname" (x)
+$plg_geb_displayNames = 1;
```

Comment:
Where do you use this setting?
Also there should be always a value, so "Nachname" should be value 2.